### PR TITLE
Time Series: Card width adjustment, connect settings and ui together

### PR DESCRIPTION
**Pull request by japie1235813**
Last piece of our card width adjustment. Take the updated value from setting panel and apply on the card css to change the width.

### CODE REVIEW

File: tensorboard/webapp/metrics/store/metrics_reducers.ts [hunk 1]
```diff
@@ -646,6 +646,24 @@ const reducer = createReducer(
       },
     };
   }),
+  on(actions.metricsChangeCardWidth, (state, {cardMaxWidthInVW}) => {
```
Comment (yatbear) [inline tensorboard/webapp/metrics/store/metrics_reducers.ts:649]: (Dunno how to comment lines above, so adding it here) On line 398, do we need to have
```
if (typeof partialSettings.timeSeriesCardMaxWidthInVW === 'number') {
      metricsSettings.timeSeriesCardMaxWidthInVW = partialSettings.timeSeriesCardMaxWidthInVW;
    }
```
like the other ones?

File: tensorboard/webapp/metrics/store/metrics_reducers_test.ts [hunk 2]
```diff
@@ -773,6 +773,34 @@ describe('metrics reducers', () => {
         HistogramMode.OVERLAY
       );
     });
+
+    it('changes metricsChangeCardWidth on metricsChangeCardWidth', () => {
```
Comment (yatbear) [inline tensorboard/webapp/metrics/store/metrics_reducers_test.ts:777]: Should the name be `changes  *cardMaxWidthInVW* on metricsChangeCardWidth`?

File: tensorboard/webapp/metrics/store/metrics_reducers.ts (hunk 1 above)
Comment (japie1235813) [reply tensorboard/webapp/metrics/store/metrics_reducers.ts:649]: Unfortunately we didn't preserve the card width setting (yet) so it's meaningless to add them here.

**Review by yatbear — APPROVED** [review]

**Comment by japie1235813** [comment]
The last piece of our card width adjustment (almost). Take the updated slider value from setting panel and apply on the card css to change the width.

**Pull request merged**